### ISSUE TITLE
Borked with python 3.10.4

### ISSUE THREAD

**Issue by aisejohan** [gerby-project/plastex#60]
The error is `AttributeError: module 'collections' has no attribute 'Callable'`

**Comment by aisejohan** [comment]
587683f75beedcd6134b1984af1c81f5124c6811 and bbd5de8c3817125416976e11fa63e40981e9f6a5 in Patrick Massot tree Tested this and this gives different errors unfortunately

**Comment by jonsterling** [comment]
This is still a problem, seems to be a blocker. Any hope of fixing this?

**Comment by PatrickMassot** [comment]
Unfortunately those errors arise in parts of plasTeX where upstream diverged very significantly from the Gerby version, since https://github.com/plastex/plastex/commit/beb89c5a0eb975ef2d2f236979bbc76aa202724d.  Originally it made a lot of sense to use a fork of plasTeX to experiment freely, but I think the stacks project should have switched back to upstream plasTeX a long time ago.

**Comment by pbelmans** [comment]
I agree getting it back to upstream would be great. Sadly, time is not an infinite resource. 

I remember getting this to work (or have quick workaround rather) when we ran into this in August. I'll take another look after the Christmas break.
